app/views/explore.scala.html

```diff
@@ -39,6 +33,21 @@
         }
     </script>
 
+    <style>
```

Comment:
Is there a CSS file that this can live in that will work?

Reply:
Yep! I'm cringing at myself right now for leaving this in here. I think it's a remenant from my testing. Fixed now by moving it to the `svl.css` file.